apps/web/src/lib/get-programmerbar-status.ts

```diff
@@ -0,0 +1,13 @@
+export async function getProgrammerbarStatus() {
+  try {
+    return (await fetch("https://api.programmer.bar").then((res) => res.json())) as {
+      success: true;
+      message: string;
+    };
```

Comment:
Denne typen blir litt feil. Den returnerer en `status` som er `number` og en `message` som er `string`. https://github.com/programmerbar/api/blob/main/src/routes/status-routes.ts#L11. Hva de forskjellige tallene på `status` betyr kan du se her: https://github.com/orgs/echo-webkom/projects/2?query=sort%3Aupdated-desc+is%3Aopen&pane=issue&itemId=59732687.

Så burde være 
```ts
{
  status: number;
  message: string;
}
```